blog: how to choose and measure growth loops (#4469)
PostHog/posthog.com: contents/blog/growth-loops.md

---
date: 2022-10-18
title: "How to choose and measure growth loops"
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ["ian-vanagas"]
featuredImage: ../images/blog/posthog-marketing/marketing-hog.jpg
featuredImageType: full
categories: ["Product analytics", "Guides"]
---

> A growth loop is a series of actions that amplify each other to fuel perpetual growth. New inputs repeatedly power the series to exponentially grow the whole. Companies use growth loops to align their product and distribution to have the greatest impact.

Reviewer: ```suggestion
A growth loop is a series of actions that amplify each other to fuel perpetual growth – new inputs repeatedly power the series to exponentially grow the whole. Companies use growth loops to align their product and distribution to have the greatest impact.
```

Growth loops are an alternative way to thinking than [funnels](/blog/aarrr-pirate-funnel). While funnels focus on a series of steps from Acquisition to Referral, growth loops focus on a smaller number of steps done repeatedly.

## Why you should care about growth loops

Growth loops are important because they compound. Ideally, the growth in your key metrics exponentially increases, while the average effort or investment decreases. For many companies, this key metric is users or revenue which helps create a successful company.

> Growth loops can be a useful way of aligning multiple areas. For a growth loop to work, customer-facing teams must align with engineering and product teams. Each team benefits from the work the others are doing. 

Author: Added Substack as an example because I didn't want to be too cliche with an Amazon or Facebook example.

For example, sales is a growth loop for enterprise SaaS companies (e.g. Salesforce, SAP, Microsoft) and big consultancies (e.g. McKinsey, Deloitte)

## Choosing the right growth loop

After reading these, you likely have some idea of what growth loop is right for your product and company. Two key questions to understand this further:

1. Who are your ideal users?
2. What benefits do those users get as part of your growth loop?

Understanding your ideal user will help you decide where to put your effort for your growth loop. If your ideal user won’t pay for your product, it isn’t a good idea to use sales as your growth loop. You can create a growth loop that better aligns with what they want. Growth loops are about creating win-wins.

A user needs a win-win to repeatedly take an action that fuels a product’s growth loop. They need an incentive. Smart companies build this incentive into the core of their product. For example, you gain followers by posting on social media, but that also keeps users engaged and attracts new users.

> The benefits your product and growth loop can provide are limitless. Finding new ways to provide this value is a great way.

Reviewer: Again, I think this section would benefit from one or two examples. Walk people through the process of choosing a growth loop, maybe using some well-known companies as theoretical examples, or just different types of companies (e.g. one B2B example, one B2C example)

Author: Added!

## Measuring your growth loop

To understand your growth loop and the value it is providing, it is important to measure your chosen growth loop. There are several ways to do this; they usually involve measuring the input (action) in connection to the output (key metric). What these inputs and outputs depend on the product and the growth loop. 

Here are some examples of how you can measure each of the growth loops.

### Virality

To measure virality, we want to know what drives users to share our product and the referral source of sign ups. To do this, we’ll want to measure our traffic and shareability. We want events for the loop of signing up, getting started, referring others, and repeating. More occurrences of each and better connections between them are good signs for our viral growth loop.

### Performance marketing

Performance marketing is about measuring marketing is leading to signups and revenue. We can do this by understanding customer cost of acquisition (CAC) and lifetime value (LTV). The margin between what it takes to acquire a user (CAC) and how much value we receive from them (LTV) drives the performance marketing growth loop (and the rest of the company).

> To get data about CAC and LTV, we’ll have to understand the sources of our traffic, conversion, and finally retention. This combined with our marketing spend and revenue enables us to calculate CAC and LTV (and grow our business and product).

Reviewer: Could we add a bit on how to calculate CAC and LTV here? Just the basic formula would be enough.

Author: Added a note

---
date: 2022-10-19
title: "How to choose and measure growth loops"
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ["ian-vanagas"]
featuredImage: ../images/blog/posthog-marketing/marketing-hog.jpg
featuredImageType: full
categories: ["Product analytics", "Guides"]
---

A growth loop is a series of actions that amplify each other to fuel perpetual growth. New inputs repeatedly power the series to exponentially grow the whole. Companies use growth loops to align their product and distribution to have the greatest impact.

> Growth loops are an alternative way to thinking than [funnels](/blog/aarrr-pirate-funnel). While funnels focus on a series of steps from Acquisition to Referral, growth loops focus on a smaller number of steps done repeatedly.

Reviewer: ```suggestion
Growth loops are an alternative growth framework to funnels like [AARRR pirate metrics](/blog/aarrr-pirate-funnel). While funnels focus on a series of steps from Acquisition to Referral, growth loops focus on a smaller number of steps done repeatedly.

In this guide, we'll cover:

- Why growth loops are so powerful
- The different types of growth loops
- How to choose your own growth loop
- How to measure your growth loop
```

> ## Why you should care about growth loops

Reviewer: ```suggestion
## Why should you care about growth loops?
```

Growth loops are important because they compound. Ideally, the growth in your key metrics exponentially increases, while the average effort or investment decreases. For many companies, this key metric is users or revenue which helps create a successful company. 

Nearly every successful company utilizes growth loops in some way, consciously or not. For example, the newsletter platform Substack allows writers to easily share and monetize their work. As writers share their work (for their own benefit), it gets the attention of other writers, who see the benefit of the platform, join, and continue the loop.

Growth loops can be a useful way of aligning multiple areas. For a growth loop to work, customer-facing teams must align with engineering and product teams. Each team benefits from the work the others are doing. 

> In our Substack example, the marketing and business development teams have to attract large writers (who often required upfront payments). The product and engineering teams then have to deliver a great writing and subscription experience. Without both, the growth loop doesn't work. When the company is aligned, however, it can cause exponential growth through compounding benefits. For Substack, that means bigger writers, better experiences, more readers, and ultimately, more revenue.

Reviewer: ```suggestion
In our Substack example, the marketing and business development teams have to attract popular writers (who often require upfront payments). The product and engineering teams then have to deliver a great writing and subscription experience. Without both, the growth loop doesn't work. When the company is aligned, however, it can cause exponential growth through compounding benefits. For Substack, that means bigger writers, better experiences, more readers, and ultimately, more revenue.
```

Author: Ok, asking Lottie.

## Types of growth loops

There are multiple types of growth loops. Each involves specific actions and is relevant to specific companies. In this section, we’ll define each of them and provide examples. After this, we’ll cover how to choose a growth loop. 

### Virality

> Virality is the growth loop of spreading rapidly through word of mouth. Users gain value from other users being a part of the product or by sharing the product. Products succeed when they gain large amounts of attention.

Reviewer: ```suggestion
Virality is all about growth through word of mouth. Users gain value from other users being a part of the product or by sharing the product. Products succeed when they gain large amounts of attention.
```

This is generally good for two types of products:

- Multiplayer or networked products that improve (or only work) when multiple friends or colleagues also use it (e.g. Snapchat, Figma, Zoom). Users want friends or colleagues to use the product.
- Products that are fun to share with others (e.g. TikTok, Instagram, Airbnb listings). Users share the app or content from the app on social media and other communications platforms.

### Performance marketing (paid ads)

The performance marketing growth loop is using advertising or marketing to acquire users, then using the new revenue to fund more marketing. It is reliant on connecting marketing to acquire users (performance) with value (revenue) captured. If you see a social media or search ad for a product, that is a product relying on performance marketing.

For example, with Google Ad Words, companies pay per click on specific keywords related to potential customers’ searches. They pay for clicks (performance). With the correct math figured out (cost of acquisition), this can lead to a growth loop of increasing ad spend and revenue growth.

> Products that benefit from performance marketing are generally high margins (more money for ads) with low organic demand. For example, direct-to-consumer goods (e.g. Casper, Warby Parker, Dollar Shave Club) and SaaS software targeting small-medium businesses (e.g. Shopify, Hubspot, Slack).

Reviewer: ```suggestion
 Products that use performance marketing growth loops generally have high margins (more money for ads), but low organic demand. Good examples include direct-to-consumer goods (e.g. Casper, Warby Parker, Dollar Shave Club) and SaaS software targeting small-medium businesses (e.g. Shopify, Hubspot, Slack).
```

What you can do is create a connection between their network and content creation. They value their network a lot, and creating content is free. When a user creates and shares content, their network pays attention. With this attention, the user can engage with their network, and LinkedIn can sell ads. When other users see someone benefiting from this, they share content, and this cycle repeats. This is a content growth loop.

A user needs a win-win to repeatedly take an action that fuels a product’s growth loop. They won't take an action they don't think they'll benefit from. They need an incentive, whether it is status, attention, or money. Smart companies build this incentive into the core of their product.

For example, HubSpot builds a large number of free tools small-medium businesses benefit from, like "Website Grader" or "Email Signature Generator" as well as the free tier of their product. These freebies get a lot of attention which drives signups, some of which turn into customers. The growth in customers allows them to improve their overall product (including their free tools) and create a B2B viral growth loop. 

> The benefits your product and growth loop can provide are limitless. Finding new ways to provide this value is a great way.

Reviewer: Last sentence is missing something!

---

> date: 2022-10-19

Reviewer: ```suggestion
date: 2022-10-31
```
